scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -6,30 +6,47 @@
 import logging
 import os
 import sys
-
+import subprocess
 from clusters import GKECluster
 from compatibility_test import make_compatibility_test_runner
 from get_latest_helm_chart_versions import get_latest_helm_chart_versions
 
+
 # set required test parameters
 os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
 os.environ["ROX_POSTGRES_DATASTORE"] = "true"
 
+central_chart_versions = get_latest_helm_chart_versions("stackrox-central-services", 2)
+sensor_chart_versions = get_latest_helm_chart_versions("stackrox-secured-cluster-services", 3)
+latest_tag = subprocess.check_output("make tag", shell=True).strip().decode("utf-8")
```

Comment:
1. [minor and optional] There's no need to `.decode("utf-8")` because I believe you can provide `"utf-8"` as encoding argument to `check_output`.

2. When you need only the output from the `Makefile` it's good to at least add `--quiet --no-print-directory` attributes. None of them is strictly required in **this** case but having them is a good habit for cases of moving code and for consistent style.

```python
subprocess.check_output("make --quiet --no-print-directory tag", shell=True)
```

3. Just calling `make tag` without specifying the directory of the `Makefile` assumes this script is executed from the repo root as the current directory. However, this specific file `scripts/ci/jobs/gke_version_compatibility_tests.py` seems to be intended as executable (look at the shebang at the line 1). As such, it can be executed from any directory and so `make tag` will fail. The robust way to handle this is to derive the repo root as N-th parent of the current `__file__` and provide it as `-C` command-line argument:
```
subprocess.check_output(["make", "-C", repo_root, "--quiet", "--no-print-directory-tag"], shell=False)...
```
Note that you'd need to deal with path-escaping if you try embed `repo_root` in the command with `shell=True`.